src/components/Review/HistoryPanel.tsx

```diff
@@ -6,7 +6,7 @@ import HistoryResponseElement from '../PageElements/Elements/HistoryResponseElem
 import { useUserState } from '../../contexts/UserState'
 import { Stage } from '../Review'
 import strings from '../../utils/constants'
-import { StageDetails } from 'utils/types'
+import { StageDetails } from '../../utils/types'
```

Comment:
Wonder why this needed changing? Was it also causing an error that we didn't notice?

Reply:
yeah was wondering the same, but didn't have time to go over it, will investigate if the problem persists